NeoML/src/Dnn/Optimization/ChannelwiseWith1x1Optimizer.cpp

```diff
@@ -64,21 +64,31 @@ int CChannelwiseWith1x1Optimizer::optimizeNonResidualBlocks()
 		graph.SelectLayer( *downConv );
 
 		CBaseLayer* channelwiseActivation = graph.SelectConnectedOutput<>( *downConv, 0 , true ).Layer;
-		if( channelwiseActivation == nullptr || !isValidActivation( *channelwiseActivation ) ) {
+		CChannelwiseConvLayer* channelwise = nullptr;
```

Comment:
You may declare `channelwise` variable after next `if`.

Reply:
Yep, fixed. Thanks!